app/views/boards/show.html.erb

```diff
@@ -18,6 +18,7 @@
         <td>
           <%= link_to 'Edit', "/boards/#{@board.id}/lists/#{list.id}/edit" %>
           <%= link_to 'Delete', "/boards/#{@board.id}/lists/#{list.id}", method: :delete, data: { confirm: "Are you sure?"} %>
+          <%= link_to 'Show', "/boards/#{@board.id}/lists/#{list.id}" %>
```

Comment:
uzywaj helperów :) to był tylko przykład zeby pokazać Ci jak to dziala. Ale normlanie normalnie uzywamy helperów czyli to co masz w rake rotues